src/Microsoft.Identity.Client/AppConfig/ConfidentialClientApplicationBuilder.cs

```diff
@@ -55,18 +58,25 @@ internal ConfidentialClientApplicationBuilder(ApplicationConfiguration configura
         }
 
         /// <summary>
+        /// Creates a ConfidentialClientApplicationBuilder from a clientID. 
+        /// See https://aka.ms/msal-net-application-configuration
         /// </summary>
-        /// <param name="clientId"></param>
-        /// <returns></returns>
+        /// <param name="clientId">Client ID (also known as App ID) of the application as registered in the
+        /// application registration portal (https://aka.ms/msal-net-register-app)/.</param>
+        /// <returns>A <see cref="ConfidentialClientApplicationBuilder"/> from which to set more
+        /// parameters, and to create a public client application instance</returns>
         public static ConfidentialClientApplicationBuilder Create(string clientId)
         {
             var config = new ApplicationConfiguration();
             return new ConfidentialClientApplicationBuilder(config).WithClientId(clientId);
         }
 
         /// <summary>
+        /// Sets the Application token cache. This cache is used in client credential flows
+        /// (<see cref="IConfidentialClientApplication.AcquireTokenForClientAsync(System.Collections.Generic.IEnumerable{string})"/>
+        /// and its override
         /// </summary>
-        /// <param name="tokenCache"></param>
+        /// <param name="tokenCache">Application token cache</param>
         /// <returns></returns>
         [Obsolete("You can access the AppTokenCache using the AppTokenCache property on the created IConfidentialClientApplication")]
         public ConfidentialClientApplicationBuilder WithAppTokenCache(TokenCache tokenCache)
```

Comment:
Since we want these to be already obsolete, I'd like to discuss if we even want/need these methods based on the impact to change in the samples.  I'd prefer not to have these at all (note they currently throw notimplex)